frontend/src/app/login/login.component.html

```diff
@@ -0,0 +1,10 @@
+<div>
+  Welcome! Please login or die
```

Comment:
I know, it's a mock, but please use a little more user-friendly phrase :joy:

Reply:
Done